parfive/tests/test_downloader.py

```diff
@@ -6,6 +6,20 @@
 from pytest_localserver.http import WSGIServer
 
 from parfive.downloader import Downloader, Token, FailedDownload, Results
+import hashlib
+
+
+def sha256sum(filename):
+    """
+    https://stackoverflow.com/a/44873382
+    """
+    h  = hashlib.sha256()
+    b  = bytearray(128*1024)
+    mv = memoryview(b)
+    with open(filename, 'rb', buffering=0) as f:
+        for n in iter(lambda : f.readinto(mv), 0):
+            h.update(mv[:n])
+    return h.hexdigest()
```

Comment:
I think that any functions added in a test suite should be tested.

Reply:
Could this also not be moved to util?

Reply:
It's not used anywhere else in the code, should we still include it in util? Yeah, a test might help. :)

Reply:
Is this function really not in the stdlib somewhere? Either way, I agree with Nabil, move it into util and test it please.